app/services/generate_membership_pdf.rb

```diff
@@ -11,6 +11,7 @@ def as_html
     member_type = 'insider' if @user.svie_post_request.member_type == SvieUser::INSIDE_MEMBER
     member_type = 'outsider' if @user.svie_post_request.member_type == SvieUser::OUTSIDE_MEMBER
     member_type = 'not_member' if @user.svie_post_request.member_type == SvieUser::NOT_MEMBER
+    member_type = 'inactive' if @user.svie_post_request.member_type == SvieUser::INACTIVE_MEMBER
```

Comment:
I think the naming is not fully consistent here, why don't we use the same 'membertype_member' names in the pdf templates?